### PR TITLE
Optimize docker for VATUSA deployment

### PR DESCRIPTION
**Pull request by dhawton**
Fixed travis and docker build files for deployment, docker-compose is production ready as well

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,26 @@
 language: node_js
 node_js:
-- 9
-cache:
-  directories:
-  - node_modules
+  - 8
```
Comment (rahulp959) [inline .travis.yml]: Why 8 and not 9? We will be using features of 9 to reduce transpilation time.

File: .travis.yml [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+env:
+  - DOCKER_COMPOSE_VERSION=1.17.1
+
+services:
+  - docker
+
+before_install:
+ - curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` > docker-compose
+ - chmod +x docker-compose
+ - sudo mv docker-compose /usr/local/bin
```
Comment (rahulp959) [inline .travis.yml]: Why do we need it in /usr/local/bin?

File: .travis.yml (hunk 1 above)
Comment (dhawton) [reply .travis.yml]: Just used what I had, easy to change.

File: .travis.yml (hunk 2 above)
Comment (dhawton) [reply .travis.yml]: Just following travis docs, part of path and ensures it is placed correctly. https://docs.travis-ci.com/user/docker/

**Pull request merged**